components/diagnose/scan.go

```diff
@@ -233,17 +234,16 @@ func Scan(ctx context.Context, opts ...OpOption) error {
 		}
 
 		fmt.Printf("%s scanning dmesg for %d lines\n", inProgress, op.lines)
-		defaultDmesgCfg, err := dmesg.DefaultConfig(ctx)
```

Comment:
so the default config has the 3 hours limit? can we remove this default config is not used anymore?

Reply:
>  dmesg.DefaultConfig(ctx)

Yes correct. This is the "old" dmesg default configuration, to be deprecated, as soon as we migrate.

>  remove this default config

will do, once we deprecate the dmesg component, pretty soon